assets/javascript/apps/pipeline/Pipeline.tsx

```diff
@@ -136,10 +146,19 @@ export default function Pipeline() {
         deleteKeyCode={[]}
         defaultEdgeOptions={defaultEdgeOptions}
         onSelectionChange={onSelectionChange}
+        onPaneClick={() => setIsOpen(false)} // Close panel when clicking on the canvas
       >
-        <Controls showZoom showFitView showInteractive position="bottom-left"/>
-        {/*<MiniMap position="bottom-right"/>*/}
-        <Background variant={BackgroundVariant.Dots} gap={12} size={1}/>
+        <SidePanel
```

Comment:
nit: Maybe we can rename this to something like `ComponentList` or something, since it is no longer on the "Side"